simulator/func_sim/alu.h

```diff
@@ -236,6 +236,13 @@ struct ALU
 
     // Bit manipulations
     template<typename I> static void sbext( I* instr) { instr->v_dst[0] = 1U & ( instr->v_src[0] >> shamt_v_src2<typename I::RegisterUInt>( instr)); }
+    
+    template<typename I> static void max( I* instr)
+    { 
+	instr->v_dst[0] = std::max<typename I::RegisterSInt>>(instr->v_src[0], instr->v_src[1]);
```

Comment:
We do not use tabs, Please configure your editor: https://github.com/MIPT-ILab/mipt-mips/wiki/Code-style-guidelines#indentation

Reply:
Please, follow our guidelines on spaces around parentheses: https://github.com/MIPT-ILab-Compilers/MIPT-Vis/wiki/Syntax#spaces